sdk/core/azure-core-http-netty/src/main/java/com/azure/core/http/netty/NettyAsyncHttpClientBuilder.java

```diff
@@ -118,10 +118,6 @@ public NettyAsyncHttpClientBuilder connectionProvider(ConnectionProvider connect
     /**
      * Sets the {@link ProxyOptions proxy options} that the client will use.
      *
-     * <p><strong>Code Samples</strong></p>
-     *
-     * {@codesnippet com.azure.core.http.netty.NettyAsyncHttpClientBuilder#proxy}
```

Comment:
is this intentional ?

Reply:
Did you read the description of this PR? :)

Reply:
okay, I thought this was for perf framework style fixes only.

Reply:
I will look into why these codesnippets have issues. Maybe using the fully qualified package name for `Type` might help disambiguate.